buidl/script.py

```diff
@@ -101,10 +104,18 @@ def parse(cls, stream):
                 commands.append(op_code)
         obj = cls(commands)
         if count != length:
+            # FIXME: shouldn't this throw an error?
```

Comment:
It used to throw an error, but that was out of consensus. Turns out that Bitcoin Core will read the number of bytes that are there and not what was promised and figure it out!

Reply:
I would take this out.